Formula/rsync.rb

```diff
@@ -52,6 +60,11 @@ def install
     # function multiversioning on older versions of macOS
     args << "--disable-simd" if MacOS.version < :catalina
 
+    # Remove in the next release along with autoconf/automake deps
+    # Needed for rsync-schar-fix-3.2.4.diff
+    system "autoreconf", "-fi"
+    system "autoconf", "-o", "configure.sh"
```

Comment:
This is only needed on ARM right?

Reply:
Yes indeed, but patch is also correct on x86.

Reply:
Seems better to only patch what is broken, yes.